spec/mailtrap/client_spec.rb

```
@@ -250,4 +250,142 @@
+          sandbox: true
+        )
+      end
+
+      it 'raises an error' do
+        expect do
+          client.send_batch(base_mail, recipients)
+        end.to raise_error(ArgumentError, 'bulk mode is not applicable for sandbox API')
+      end
+    end
+
+    context 'in bulk mode' do
```

RuboCop (1.75.5)

[convention] Context description should match /^when\b/, /^with\b/, or /^without\b/.

(RSpec/ContextWording)